src/messaging/ExchangeDelegate.h

```diff
@@ -77,22 +77,20 @@ class DLL_EXPORT ExchangeDelegateBase
      */
     virtual void OnExchangeClosing(ExchangeContext * ec) {}
 
-    virtual ExchangeMessageDispatch * GetMessageDispatch(ReliableMessageMgr * rmMgr, SecureSessionMgr * sessionMgr) = 0;
-};
-
-class DLL_EXPORT ExchangeDelegate : public ExchangeDelegateBase
-{
-public:
-    virtual ~ExchangeDelegate() {}
-
+    /**
+     * @brief
+     *   This function returns the message dispatch that the exchange context should use
+     *   to transmit messages generated by the exchange.
+     *
+     *  @param[in]    rmMgr         Reliable message manager object pointer
```

Comment:
nit: delete redundant doccomment bits. telling me that `ReliableMessageMgr * rmMgr` is "a reliable message manager object pointer" seems not to help me as much when understanding the code and has the downside of using up vertical space in the editor.